### PR TITLE
Azure RG Task cosmetic changes

### PR DESCRIPTION
**Pull request by kuvinodms**

### CODE REVIEW

File: Tasks/AzureResourceGroupDeployment/main.ts [hunk 1]
```diff
@@ -15,7 +15,7 @@ function run(): Promise<void> {
                 return resourceGroupOperationsController.createOrUpdateResourceGroup();
             case "DeleteRG":
                 return resourceGroupOperationsController.deleteResourceGroup();
-            case "Select Resource Group":
+            case "Configure deployment options for vitual machine":
```
Comment (bishal-pdMSFT) [inline Tasks/AzureResourceGroupDeployment/main.ts]: You will need to handle back-compat. "Select Resource Group" value should still work for release definitions which were created earlier.

File: Tasks/AzureResourceGroupDeployment/task.json [hunk 2]
```diff
@@ -34,7 +34,7 @@
             "name": "AdvancedDeploymentOptions",
             "displayName": "Advanced deployment options for virtual machines",
             "isExpanded": true,
-            "visibleRule": "action = Create Or Update Resource Group || action = Select Resource Group"
+            "visibleRule": "action = Create Or Update Resource Group || action = Configure deployment options for virtual machine"
```
Comment (bishal-pdMSFT) [inline Tasks/AzureResourceGroupDeployment/task.json]: handle compat here as well if needed

File: Tasks/AzureResourceGroupDeployment/README.md [hunk 3]
```diff
@@ -45,7 +45,7 @@ The parameters of the task are described in details, including examples, to show
  
  * **Action**\*: Select the action to be performed on the resource group. Following actions are available:
     - Create or Update Resource Group: creates a new resource group or to update an existing one (using [Azure Resource Manager templates](https://azure.microsoft.com/en-in/documentation/articles/resource-group-template-deploy/)).
-    - Select Resource Group: reads the Resource Group information from Azure and creates an Output Variable with the VMs in the Resource Group. Read more about Output Variables [here](#output-variables).
+    - Configure deployment options for virtual machine: reads the Resource Group information from Azure and creates an Output Variable with the VMs in the Resource Group. Read more about Output Variables [here](#output-variables).
```
Comment (bishal-pdMSFT) [inline Tasks/AzureResourceGroupDeployment/README.md]: Change the description based on new behavior - focus on option to configure deployment option

**Comment by kuvinodms** [comment]
@bishal-pdMSFT 
Instead of keeping "Select Resource Group" in code will induce confusion in future as we have removed the option from the action list. Hence I'm updating only text corresponding to "Select Resource Group" in order to maintain consistency and avoid potential scope of regression.

**Pull request merged**